src/main-page.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable import/no-cycle */
+import { createNavigation, addAllEventListeners } from './nav-creator';
+/* eslint-enable import/no-cycle */
+
+function createMainPage() {
+  const content = document.getElementById('content');
+
+  content.innerHTML = '';
+
+  createNavigation();
+
+  const restaurantTitle = '<h1>Restaurant name</h1>';
+  const restaurantImage = '<img src="" alt="image of Restaurant">';
+  const restaurantDescription = '<p>This is one of the best Restaurant that I had the privilege of eating in</p>';
```

Comment:
- [ ] [required] Please use `document.createElement` and `document.appendChild` to create and append HTML elements to the `<div>` with `id=content`. This is an opportunity for you to practice/explore building HTML elements using only javascript. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/append_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/restaurant-page))